src/main/webapp/templates/profileBuilder.html

```diff
@@ -9,7 +9,8 @@
   <link href="css/styles.css" rel="stylesheet">
 </head>
 <body>
-  
-  <script src="ts/app.ts"></script>
+
+
+  <script src="ts/profileBuilder.ts"></script>
```

Comment:
this looks like an unrelated change from the tests? Can you split this into a separate CL?

Reply:
yes, sorry about that